services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/attachment/common/AttachmentContentEntity.kt

```diff
@@ -36,7 +37,13 @@ class ApplicationAttachmentContentEntity(
 ) : AttachmentContentEntity(attachmentId, content)
 
 @Repository
-interface HankeAttachmentContentRepository : JpaRepository<HankeAttachmentContentEntity, UUID>
+interface HankeAttachmentContentRepository : JpaRepository<HankeAttachmentContentEntity, UUID> {
+
+    @Query(value = "SELECT * FROM hanke_attachment_content LIMIT 1", nativeQuery = true)
+    fun pickOne(): HankeAttachmentContentEntity?
+
+    fun deleteByAttachmentId(attachmentId: UUID)
```

Comment:
Since attachmentId is the primary key, this is the same as the built-in `deleteById(ID id)`.

Reply:
Oh, ok I did not see that. Thx.